src/subscription-user/subscription-user.controller.ts

```diff
@@ -44,8 +45,8 @@ export class SubscriptionUserController {
     @Body() updateSubscriptionsDto: UpdateSubscriptionUserDto,
   ): Promise<ISubscriptionUser | undefined> {
     return this.subscriptionUserService.cancelWhatsappSubscription(
-      req['user'].user.id,
-      req['user'].user.email,
+      req['userEntity'].user.id,
+      req['userEntity'].user.email,
```

Comment:
```suggestion
      req['userEntity'].id,
      req['userEntity'].email,
```

the new `userEntity` object doesn't have a child `user` object

Reply:
Ahh yes apologies, I have changed that now